src/org/intellij/erlang/runconfig/ErlangRunConfigurationBase.java

```diff
@@ -71,6 +82,20 @@ public void writeExternal(Element element) throws WriteExternalException {
   public void checkConfiguration() throws RuntimeConfigurationException {
     ErlangModuleBasedConfiguration configurationModule = getConfigurationModule();
     configurationModule.checkForWarning();
+    checkWorkDirectory();
+  }
+
+  private void checkWorkDirectory() throws RuntimeConfigurationError {
+    if (myWorkDirectory != null && !myWorkDirectory.isEmpty()) {
+      File dir = new File(myWorkDirectory);
+      try {
+        if (!dir.isDirectory()) {
+          throw new RuntimeConfigurationError("Incorrect path to working directory.");
+        }
+      } catch (SecurityException e) {
```

Comment:
We don't have ```'catch' on newline``` code style option enabled in this repo. May be we should enable it, I find having having a newline before ```catch``` more convenient, too.